clamd.go

```diff
@@ -377,14 +377,16 @@ func (c *Client) streamCmd(tc *textproto.Conn, cmd protocol.Command, f io.Reader
 
 	fmt.Fprintf(tc.W, "n%s\n", cmd)
 	b := make([]byte, 4)
-	for {
+	eof := false
```

Comment:
I don't think you should be initializing a local variable for each loop run, the eof should move outside the for scope